components/shared/ui/menu.tsx

```diff
@@ -60,7 +60,8 @@ export const Menu: FC<Props> = props => {
         <div className="w-screen h-full md:flex justify-between z-50">
           <div className="flex h-full justify-between items-center px-8 py-4">
             <Link href="/" className="flex items-center">
-              <span className="font-extrabold">HealthTech</span>
+              <span className="font-extrabold">Ficto</span>
```

Comment:
It would be great to see the different names based on the site they are used on.